bin/build/test/release/version_info_test.clj

```diff
@@ -17,34 +17,43 @@
    :patch      patch
    :highlights highlights})
 
+;; TODO -- can result in a flakey test if the date changes during this test run
 (def ^:private today (.format (DateTimeFormatter/ofPattern "yyyy-MM-dd") (LocalDate/now)))
 
-(def ^:private test-versions [["39.0" today false ["Super New Fix 1" "Super New Fix 2"]]
-                              ["38.3" "2021-04-01" true ["Fix 1" "Fix 2"]]
-                              ["38.2" "2021-03-21" true ["Older Fix 1" "Older Fix 2"]]
-                              ["38.1" "2021-03-14" true ["Much Older Fix 1" "Much Older Fix 2"]]
-                              ["38.0" "2021-02-15" true ["Super Old Fix 1" "Super Old Fix 2"]]])
+(def ^:private test-versions
+  [["39.0" today false ["Super New Fix 1" "Super New Fix 2"]]
+   ["38.3" "2021-04-01" true ["Fix 1" "Fix 2"]]
+   ["38.2" "2021-03-21" true ["Older Fix 1" "Older Fix 2"]]
+   ["38.1" "2021-03-14" true ["Much Older Fix 1" "Much Older Fix 2"]]
+   ["38.0" "2021-02-15" true ["Super Old Fix 1" "Super Old Fix 2"]]])
```

Comment:
This is fairly subjective they were here before, but I would have expect these private defs to be inside `build-info-test`. Same with `previous-version-cases`. The main benefit is readability - these defs are very specific to the test's implementation and should be grouped with the assertions in a `deftest` block.

Reply:
That makes sense, although one concern I have is that the deftest expression will balloon somewhat! 

I'm still adjusting to reading Lisp again, and in particular the Metabase flavour, but will take a look at this restructuring and see how it feels to me.